docs/reviewing-a-pull-request.md

```diff
@@ -0,0 +1,70 @@
+# Reviewing a Pull Request
+
+This guide is about reviewing pull requests on Exercism language tracks.
+
+## Philosophy
+
+* **Respond Kindly** We want every contributor to be glad they are here, even if we don't ultimately merge their pull request.
+* **Respond Quickly** Promptness encourages frequent, high-quality contributions from community members and other maintainers.
+* **Resolve Quickly** A pull request should either be actively in-progress, or it should be merged or closed.
+
+Closing pull requests kindly is not easy. Two great articles about this are:
+
+* [Kindly Closing Pull Requests](https://github.com/blog/2124-kindly-closing-pull-requests) by Mike McQuaid
+* [The Art of Closing](https://blog.jessfraz.com/post/the-art-of-closing/) by Jessie Frazelle
+
+## Reviewing an Exercism Exercise
+
+When reviewing a pull request, always remember the past. That is, remember that numerous users have already submitted an exercise and any significant change to an exercise invalidates their submissions to some extent.
```

Comment:
> how do you hope it will inform reviewers' actions

That's a really good question. I hope that when this happens, we should mention it out loud son that we can discuss whether or not we think it's fine in this case.